doc/source/examples/04_applied/01_naca_airfoils.mystnb

```diff
@@ -0,0 +1,253 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+# Applied: Creating a NACA 4-digit airfoil
+
+NACA airfoils are a series of airfoil shapes for aircraft wings developed by
+the National Advisory Committee for Aeronautics (NACA). They are a standardized
+system of airfoil shapes that are defined by a series of digits. The digits
+define the shape of the airfoil and are used to create the airfoil shape.
```

Comment:
```suggestion
system of airfoil shapes that are defined by a series of digits. The digits,
which indicate the shape of the airfoil, are used to create the airfoil shape.
```
Suggested edit only to add variety to the sentence structure (so the paragraph didn't read like it was written by a bot).